.github/workflows/lint-spec.yml

```diff
@@ -1,21 +1,20 @@
-name: Vacuum open API linting 
+name: Vacuum OpenAPI Linting
 run-name: Linting OpenAPI spec with Vacuum
 on: [push]
 jobs:
   vacuum-lint:
     runs-on: ubuntu-24.04
     container: dshanley/vacuum:v0.13.1
     steps:
-    - uses: actions/checkout@v4
-    - name: Generate Vacuum HTML Reports
-      run: mkdir -p reports; for FILEPATH in specs/ccapi*.json; do FILE="$(echo $FILEPATH  | cut -d '/' -f2 | cut -d '.' -f1)"; vacuum html-report "${FILEPATH}" "reports/report-${FILE}.html"; done
-      shell: bash
-    - name: Archive Vacuum HTML Reports
-      uses: actions/upload-artifact@v3
-      with:
-        name: Vacuum HTML Reports
-        path: reports/*.html
-    - name: Lint with Vaccum
-      run: vacuum lint specs/ccapi* || true
-      shell: bash
-
+      - uses: actions/checkout@v4
+      - name: Generate Vacuum HTML Reports
+        run: mkdir -p reports; for FILEPATH in specs/ccapi*.json; do FILE="$(echo $FILEPATH  | cut -d '/' -f2 | cut -d '.' -f1)"; vacuum html-report "${FILEPATH}" "reports/report-${FILE}.html"; done
+        shell: bash
+      - name: Archive Vacuum HTML Reports
+        uses: actions/upload-artifact@v3
+        with:
+          name: Vacuum HTML Reports
+          path: reports/*.html
+      - name: Lint with Vaccum
+        run: vacuum lint specs/ccapi* || true
```

Comment:
I know this is just reformatted but I think a short comment here about why we are ignoring failures may be helpful to future reviewers of this. Also on line 18 please change `Vaccum` to `Vacuum`.